### DISCUSSION TITLE
Can't login with my domain name in the mobile app

### DISCUSSION THREAD

**Discussion by smileyface24** [immich-app/immich#8234] [Q&A]
### The bug

Hello to all,

Can't login with my domain name in the mobile app

Only allow me login with my local address. Ex: 10.0.0.10:2020

But not with my domain name.

Please see the captures.

Thanks.

![IMG_0650](https://github.com/immich-app/immich/assets/123712682/8bcbc5f2-08fa-4ac4-a62b-ec5413e57139)
![IMG_0649](https://github.com/immich-app/immich/assets/123712682/dfb57486-1a7c-4431-8297-d498c39d2eb8)
![IMG_0646](https://github.com/immich-app/immich/assets/123712682/bcce5a1f-e0f4-41da-a92d-b07f2aa2358c)

### The OS that Immich Server is running on

IOS

### Version of Immich Server

lastet

### Version of Immich Mobile App

lastet

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
-
```

### Your .env content

```Shell
-
```

### Reproduction steps

```bash
-
```

### Additional information

-

**Comment by jrasm91** [comment]
Does the web work? Can you put your mobile app logs?

**Reply by smileyface24** [reply]
@jrasm91 Thanks for reply...

Look in the app mobile, only work with the local address.

But can't login with my domain name...

For that I share the capture with you guys...

And in the other hand via browser in my computer and mobile...  
(VIA WEB work well)

I can login normally not problems at all.

Finally... the logs of my mobile apps.... show me nothing.... 

Look the capture please.

Any ideas ???

I just want to login in my mobile app... with my domain name.

Thanks.

![IMG_0651](https://github.com/immich-app/immich/assets/123712682/65b41b66-f702-4862-ab95-5abcc0ef1404)

**Reply by alextran1502** [reply]
Are you using self signed certificate for ssl?

**Reply by smileyface24** [reply]
@alextran1502 Thank you for reply...

Look I don't know well... exactly.

But I will tell you a few things

1.- My domain is in Cloudflare.

2.- My Immich is hosted in UNRAID system. 

3.- Finally I can open via WEB my immich with my domain name thanks to this app: Unraid-Cloudflared-Tunnel 

I am not an expert, but I think if work well via WEB, also must work via APP mobile.

But in my particular case.... only work well via WEB because via app mobile don't let me login.

Any help is appreciate it.

Thanks.

`P.S. By the way.... I try with and without (enable and disable) the following option: "Allow self-signed SSL certificates....
But nothing happens`

**Reply by alextran1502** [reply]
From the screenshot you provided, you have gotten passed the first step which entering the url of the server, which means your can already communicate with the server over the domain name. What is wrong here is either your email or password

**Reply by smileyface24** [reply]
@alextran1502 Hello and thanks for responding again.

But I must tell you that it is not that...And why am I sure of that?

Because when entering via the web.... Example: 10.10.10.10:2020 or also mydomain.com with the same email and password. It works without any problem.

But in the phone app... on my iPhone.

It only lets me log in with the local address, not the domain name.

What's more, I use a password manager and it automatically fills in my email and password... so there's no way to go wrong.

But to test the email and password theory...

Change my password for a super easy and short one... example: Underworld1234.

With same email. Then I wrote everything manually and checked to see if everything was correct.

Everything was correct... But it gives me the same error.

`Error logging you in, check server URL, email and password`

As I said before... I don't know much... But maybe the error comes from somewhere else.
In any case, I look forward to your comments.

Thank you.

**Reply by alextran1502** [reply]
I think it will be the Cloudflare tunnel configuration since it is the only middleman in this case. I don't use CF personally for this purpose, so I suggest you come to Discord and ask the community for help

**Reply by smileyface24** [reply]
@alextran1502 Okay. Thanks. I will do that. 
Hope there someone can help me. 
Till the next time.
Regards.
